src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -95,14 +117,55 @@ module.exports.commands = [{
         this.expect.element('@code').to.not.be.present.before(1000);
 
     },
+    validateGraph: function () {
+        var self = this;
+        self.waitForElementVisible('@highlightedGraph');
+        return self;
+    },
+    validateSteps: function (browser) {
+        var self = this;
+        self.waitForElementVisible('@logConsole');
+        browser.elements('css selector', '.logConsole', function (codeCollection) {
```

Comment:
I only say it because it's adding weight to the API and would be nice to not have to pass it.